docs/ref/files/storage.txt

```diff
@@ -11,7 +11,17 @@ Django provides convenient ways to access the default storage class:
 
 .. data:: storages
 
-    Storage instances as defined by :setting:`STORAGES`.
+    Mapping of aliases to storage instances as defined by :setting:`STORAGES`.
```

Comment:
```suggestion
    A dictionary-like object that allows you to retrieve a storage instance
    using its alias as defined by :setting:`STORAGES`.
```
This is roughly the wording used for connections. I think I like "A dictionary-like object" more than "Mapping" here

Reply:
Yep, no problem. 👍